lib/LANraragi/Controller/Api/Search.pm

```diff
@@ -71,8 +71,8 @@ sub handle_api {
     my $start     = $req->param('start');
     my $sortkey   = $req->param('sortby');
     my $sortorder = $req->param('order');
-    my $newfilter = $req->param('newonly');
```

Comment:
Did you need to make this change? It's fine btw, just curious

Reply:
The backend just kept throwing me tons of "Use of uninitialized value $untaggedf in string eq".
After setting a default values, these annoying errors didn't came to me on my test server. That's it.